src/main/java/org/aksw/simba/lemming/mimicgraph/metricstorage/ConstantValueStorage.java

```diff
@@ -232,4 +234,34 @@ public void loadData(){
 	public static String generateGraphKey(ColouredGraph graph) {
 	    return graph.getGraph().getNumberOfVertices() + "-" + graph.getGraph().getNumberOfEdges();
 	}
+	
+	/** The method returns the list of metrics which are present in characteristics expressions.
+	 * @param lstMetrics - The list which contains all the input metrics.
+	 * @return List of metrics
+	 */
+	public List<SingleValueMetric> getMetrics(List<SingleValueMetric> lstMetrics){
```

Comment:
The name of the method is not really expressive. `getX` typically means that the class knows X and gives it to you. However, in this case, you already have the list that includes X and want a shorter, filtered list. So maybe `filterMetrics` or `getMetricsOfExpressions` would be better. :thinking:

Reply:
Updated the method name to getMetricsOfExpressions.